extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -269,3 +270,79 @@
 .hidden {
 	display: none;
 }
+
+#WikiaArticle {
+	overflow: visible;
+}
+
+.language-choice {
+	margin: 48px 0 48px 0;
+
+	#language-choice__header {
+		color: $wds-color-dark-gray;
+		font-size: 16px !important;
+		font-weight: bold !important;
+		line-height: 1.5 !important;
```

Comment:
now, after merging with feature branch we don't need this selector at all :) removed